pins/tests/test_boards.py

```diff
@@ -304,6 +310,47 @@ def test_board_pin_read_insecure_succeed_board_flag(board):
         board.pin_read("test_pin")
 
 
+# pin_write with unversioned boards ===========================================
```

Comment:
testing the following:

- if a BOARD is unversioned
 	- first write: okay
 	- following writes:
 	   - if PIN is versioned: keep all copies
 	   - if PIN is unversioned: keep latest copy
 	   - if PIN is not specified: keep latest copy

- if a BOARD is versioned
 	- first write: okay
 	- following writes:
 	   - if PIN is versioned: keep all copies (as normal)
 	   - if PIN is unversioned: raise error
 	   - if PIN is not specified: keep all copies (as normal)

Reply:
Thanks for this note, it's super handy!